app/javascript/packs/base.jsx

```diff
@@ -5,6 +5,40 @@ import {
   initializeTouchDevice,
 } from '../topNavigation/utilities';
 
+window.presentModal = async (
```

Comment:
Refactor (blocking): As mentioned above, this would provide better DX if it were a parameter object.